linode/linode-cli
# fix: Resolve help page error for list commands with non-standard names

## 📝 Description

This change resolves an error that would occur when attempting to view the help page for certain commands with non-standard action names (e.g. `domains records-list --help`). This issue occurred because the branch condition for list commands explicitly checked for an action with the name `list` which isn't met by some list commands, leading to [this code path](https://github.com/linode/linode-cli/blob/dev/linodecli/arg_helpers.py#L392-L417) being run and raising an error.

Resolves #581 

## ✔️ How to Test

The following test steps assume you have pulled down this PR locally and run `make install`.

### Integration Testing

```
make INTEGRATION_TEST_PATH=domains/test_domain_records.py testint
```

### Manual Testing

1. Run the following command:

```
linode-cli domains records-list --help
```

2. Ensure the help page prints as expected with no errors.
